test/index.spec.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  parseLockFile,
```

Comment:
Please can we follow our test naming standards of `name.test.ts`? `test/index.spec.ts` => `test/index.test.ts`

Reply:
changed